pom.xml

```diff
@@ -74,7 +74,8 @@
     </version.com.ahome-it.lienzo.tooling.nativetools>
     <version.com.ahome-it.lienzo.tooling.common>1.0.190-RELEASE</version.com.ahome-it.lienzo.tooling.common>
     <version.com.allen-sauer.gwt.dnd>3.3.3</version.com.allen-sauer.gwt.dnd>
-    <version.com.fasterxml.jackson>2.9.9</version.com.fasterxml.jackson>
+    <version.com.fasterxml.jackson.bom>2.9.10.20200223</version.com.fasterxml.jackson.bom>
+    <version.com.fasterxml.jackson>2.9.10</version.com.fasterxml.jackson>
```

Comment:
I would recommend to add a comment here explaining where this property is used and why it must not be removed. It appears to be unused so without the explanation, one could remove it mistakenly or unnecessarily spend time looking for the usage.